css-layout-classes/index.html

```diff
@@ -0,0 +1,72 @@
+    <title>css-layout</title>
+  </head>
+  <body>
+    <h1>Choose your character!</h1>
+    <div class="container">
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="link">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros.<br>Ultimate</h2>
+          <p>Super Smash Bros. Ultimate<br>received universal acclaim from bothcritics and players, with some critics
+            calling it the best installment in theseries. It received praise for its large
+            amount of content and fine-turning of existing Smash gameplay elements, although
+            its online mode was widely criticized.</p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="yoshi">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-quarter">
+          <img src="images/kirby.png" alt="kirby" width="200" height="200">
+        </div>
+        <div class="column-one-quarter">
+          <img src="images/fox.png" alt="fox" width="200" height="200">
+        </div>
+        <div class="column-one-quarter">
+          <img src="images/pikachu.png" alt="pikachu" width="200" height="200">
+        </div>
+        <div class="column-one-quarter">
+          <img src="images/ness.png" alt="ness" width="200" height="200">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column2">
+          <h3>Donkey Kong</h3>
+          <p>Donkey kong is a series of video games featuring the adventures of a<br>
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981. The<br>
+            franchise consists mainly of two games genres, but also includes additional<br>
+            spin-off titles of various genres.</p>
+          <p>The games of the first genre are mostly single-screen platform/action<br>
+            puzzle types, featuring Donkey Kong as the opponet in an industrial<br>
+            construction setting.</p>
+        </div>
+        <div class="column2">
+          <img src="images/dk.png" alt="donkey kong">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column3">
+          <img src="images/mario.png" alt="mario">
+        </div>
+        <div class="column3">
+          <h4>Mario</h4>
+          <p>Mario is a fictional character in the Mario video game franchise, owned by<br>
+            Nintendo and created by Japanese video game designer Shigeru<br>
+            Miyamoto. Acting as the company's mascot, as well as being the<br>
+            eponymous protagonist of the series, Mario has appeared in over 200<br>
+            video games since his creation.</p> <p>Depicted as a short, pudgy, Italian plumber
+      who resides in the Mushroom<br>Kingdom, his adventures generally center upon rescuing
+      Princess Peach<br>from the Koopa villain Bowser.
+          </p>
+        </div>
+      </div>
```

Comment:
Did you mean to make these column-one-third and column-two-thirds?

Reply:
Yes